docs/plugin-proposal-json-modules.md

```diff
@@ -0,0 +1,89 @@
+---
+id: babel-plugin-proposal-json-modules
+title: "@babel/plugin-proposal-json-modules"
+sidebar_label: json-modules
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Transforms `import ... with { type: "json" }` declarations to platform-specific API to read and then `JSON.parse` the imported file.
+
+The transformation applied by this plugin depends on your top-level [`targets`](https://babeljs.io/docs/options#targets) to detect whether the generated code should be compatible with Node.js, browsers, or both. When targeting Node.js, the generated code will also change depending on whether you are compiling modules to CommonJS or not.
+
+:::caution
+This plugin cannot be used when compiling modules to AMD, SystemJS, or UMD.
+:::
+
+:::caution
+This plugin only transforms import decalarations and not dynamic `import()` calls.
+:::
+
+## Example
+```js title="input.js"
+import data from "./data.json" with { type: "json" };
+```
+
+will be transformed to
+
+<Tabs>
+  <TabItem value="browsers" label="Browsers" default>
```

Comment:
Since the preface mentions that "whether the generated code should be compatible with Node.js, browsers, or both", can you add a tab here to show the output when targeting both platforms?